Makefile

```diff
@@ -55,7 +55,7 @@ ${COVERAGE}: test
 integ-test: packr-build run-integ-test packr-clean
 
 run-integ-test:
-	go test -v -run Integration -tags integration ${PACKAGES}
+	go test -v -count=1 -timeout 15m -run Integration -tags integration ${PACKAGES}
```

Comment:
nit: What do you think about adding a comment saying that -count=1 is used for disabling test caching?

I always have to look up what that flag does :/